frontend/pages/projects/[id].tsx

```diff
@@ -10,8 +11,13 @@ const newProject: IProject = {
 }
 
 const ProjectDetails = (): JSX.Element => {
+
     const { projects } = useProjects()
     const [currentProject, setCurrentProject] = useState<IProject>(() => newProject)
+    const [ currentStartDate, setCurrentStartDate] = useState(new Date());
```

Comment:
Das ist nicht richtig. Start und Enddate sind bereits im currentProject enthalten. Es ist falsch, diese nochmal rauszuziehen in einen eigenen State.

Reply:
Aber ich kann nicht nur einen State haben, da setCurrentProject nur das Projekt setten kann, nicht aber das Datum